packages/web-app-user-management/src/components/Users/UsersList.vue

```diff
@@ -125,13 +138,27 @@ export default {
       ]
     },
     data() {
-      return this.orderBy(this.users, this.sortBy, this.sortDir === 'desc')
+      const orderedUsers = this.orderBy(this.users, this.sortBy, this.sortDir === 'desc')
+      return this.filter(orderedUsers, this.searchTerm)
     },
     highlighted() {
       return this.selectedUsers.map((user) => user.id)
     }
   },
   methods: {
+    filter(users, searchTerm) {
+      if (!searchTerm) {
+        return users
+      }
+      const usersSearchEngine = new Fuse(users, {
+        includeScore: true,
+        useExtendedSearch: true,
+        threshold: 0.3,
+        keys: ['displayName', 'mail', 'onPremisesSamAccountName']
```

Comment:
Would also make sense to filter by role names if possible

Reply:
Will be done later